cmd/bosun/conf/conf.go

```diff
@@ -1080,6 +1082,8 @@ func (c *Conf) loadNotification(s *parse.SectionNode) {
 			n.Get = get
 		case "print":
 			n.Print = true
+		case "contentType":
+			n.ContentType = v
```

Comment:
Would recommend against as custom mime types are possible. So long as we default to a good value let the user pick whatever they want.